ONSdigital/eq-author
# 273 / Rename Dahl to Author

### What is the context of this PR?

[Trello](https://trello.com/c/9TiWCN6e/273-change-product-name-dahl-to-author-05)

- swapped logo
- renamed "Dahl" to "Author" in relevant code that deals with meta title

### How to review 

- check logo has changed
- ensure page titles no longer contain "Dahl"
